protected/humhub/modules/space/controllers/CreateController.php

```diff
@@ -126,7 +127,11 @@ public function actionModules($space_id)
         if (count($space->getAvailableModules()) == 0) {
             return $this->actionInvite($space);
         } else {
-            return $this->renderAjax('modules', ['space' => $space, 'availableModules' => $space->getAvailableModules()]);
+            $availableModules = $space->getAvailableModules();
```

Comment:
Why not trigger an event (or use predefined one) here and handle it elsewhere, we must not interfere a module logic with the core

Reply:
- We have already changes on the core module so i don't see why only this change is making a problem here 
- To trigger an event we need to have changes on the core too 
- There's no predefined event when rendering ajax response for this action